plugin-colorpicker/colorpicker.cpp

```diff
@@ -25,81 +25,268 @@
  *
  * END_COMMON_COPYRIGHT_HEADER */
 
+
 #include "colorpicker.h"
+#include <QApplication>
+#include <QClipboard>
+#include <QDesktopWidget>
+#include <QBoxLayout>
+#include <QMenu>
 #include <QMouseEvent>
-#include <QHBoxLayout>
+#include <QPainter>
 #include <QScreen>
+#include <QSvgRenderer>
+#include <QToolButton>
+#include <XdgIcon>
+#include <lxqtgridlayout.h>
+
+
+const QString ColorPickerWidget::svgIcon {
+    "<svg xmlns=\"http://www.w3.org/2000/svg\" viewBox=\"0 0 16 16\">"
+    "  <path"
+    "     style=\"fill:%1;fill-opacity:1;stroke:none\""
+    "     d=\"M 8,2 A 6,6 0 0 0 2,8 6,6 0 0 0 8,14 6,6 0 0 0 14,8 6,6 0 0 0 8,2 M 8,3 A 5,5 0 0 1 13,8 5,5 0 0 1 8,13 5,5 0 0 1 3,8 5,5 0 0 1 8,3\""
+    "     id=\"path2\" />"
+    "  <path"
+    "     style=\"fill:%2;stroke-width:0.03125\""
+    "     d=\"M 7.3252447,12.939477 C 4.22976,12.490771 2.2993447,9.4012904 3.2660519,6.4430523 3.7960078,4.8213248 5.176118,3.5502752 6.8340753,3.1569853 8.2327213,2.8252076 9.7084288,3.1079402 10.87908,3.9319726 c 0.351893,0.2477004 0.938414,0.8342222 1.186115,1.1861147 0.392347,0.5573834 0.693227,1.2608044 0.828674,1.9373428 0.09455,0.4722872 0.101172,1.3507353 0.01371,1.8196175 -0.12871,0.6899891 -0.443837,1.4407734 -0.842381,2.0069604 -0.247701,0.351892 -0.834222,0.938414 -1.186115,1.186115 -0.549288,0.386648 -1.2303943,0.679168 -1.9132103,0.82168 -0.4104575,0.08567 -1.2226672,0.110259 -1.640625,0.04967 z\""
+    "     id=\"path32\" />"
+    "</svg>"
+};
 
 
-ColorPicker::ColorPicker(const ILXQtPanelPluginStartupInfo &startupInfo) :
-    QObject(),
-    ILXQtPanelPlugin(startupInfo)
+ColorPicker::ColorPicker (const ILXQtPanelPluginStartupInfo &startupInfo) :
```

Comment:
Please, follow the coding style... don't put the space between identifier and opening parenthesis.